### ISSUE TITLE
Forgot password

### ISSUE THREAD

**Issue by Idontker** [amosproj/amos2022ss04-digital-identity#131]
## User story

1. As a HR employee 
2. I want / need to reset my password
3. So that I can change my password if I forgot it. 

## Acceptance criteria

- "Forgot password" link on login page
- open card/pop-up to enter the email 
- send a email 
- email should be similar/contain the same functionalities as described in #128 and #129

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release
